examples/bri-3/src/bri/transactions/api/dtos/request/updateTransaction.dto.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { plainToInstance } from 'class-transformer';
+import { validate } from 'class-validator';
+import { UpdateTransactionDto } from './updateTransaction.dto';
+
+describe('UpdateTransactionDto', () => {
+  it('should return error in case payload not provided.', async () => {
+    // Arrange
+    const dto = { signature: '2323' };
+    const updateTransactionDto = plainToInstance(UpdateTransactionDto, dto);
+
+    // Act
+    const errors = await validate(updateTransactionDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('payload');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'payload should not be empty',
+    );
+  });
+
+  it('should return error in case signature not provided.', async () => {
+    // Arrange
+    const dto = { payload: 'this is a description' };
+    const updateTransactionDto = plainToInstance(UpdateTransactionDto, dto);
+
+    // Act
+    const errors = await validate(updateTransactionDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('signature');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'signature should not be empty',
```

Comment:
can we read this from constants file to avoid code change if this string is modified in actual code

Reply:
Since it is a slightly bigger change and the validation messages will change soon because we will revisit validaton as we are done with simple CRUD, i would leave it for the future. Is it ok if i open an issue for it for now?

Reply:
I'm fine a new issue..but I didn't understand why is it a big change to import the constant file and read from there?

Reply:
When the project is in it's early stages, i like to deffer decisions like these until there is a clearer picture of the exact needs. In other words, the dataset we have it to narrow to understand exactly what needs to be generalized, hence my proposal to open an issue and tackle it a later stage.

I introduced a constant file for this validation message. Please have a look.